presto-native-execution/scripts/release-centos-dockerfile/opt/entrypoint.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -eExv -o functrace
+
+SCRIPT_DIR=$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")
+PRESTO_HOME="${PRESTO_HOME:-"/opt/presto/"}"
```

Comment:
You are adding the slash everywhere. I think is better to remove it in order to avoid things like:
```
$ cat /opt/presto//config.properties.template
```

```suggestion
PRESTO_HOME="${PRESTO_HOME:-"/opt/presto"}"
```